tests/e2e/tests/build/chunk-hash.ts

```diff
@@ -0,0 +1,56 @@
+import {oneLine} from 'common-tags';
+import * as fs from 'fs';
+
+import {ng} from '../../utils/process';
+import {writeFile} from '../../utils/fs';
+import {addImportToModule} from '../../utils/ast';
+
+
+export default function() {
+  const oldHashes: {[module: string]: string} = {};
+  const newHashes: {[module: string]: string} = {};
+  // First, collect the hashes.
+  return Promise.resolve()
+    .then(() => ng('generate', 'module', 'lazy', '--routing'))
+    .then(() => addImportToModule('src/app/app.module.ts', oneLine`
+      RouterModule.forRoot([{ path: "lazy", loadChildren: "./lazy/lazy.module#LazyModule" }])
+    `, '@angular/router'))
+    .then(() => ng('build', '--prod'))
+    .then(() => {
+      fs.readdirSync('./dist')
+        .forEach(name => {
+          if (!name.match(/(main|inline|style|\d+)\.[a-z0-9]+\.bundle\.js/)) {
```

Comment:
The styles bundle should be `dist/styles.[hash].bundle.css` instead of `dist/style.[hash].bundle.js`.